sys/include/hashes/psa/riot_hashes.h

```diff
@@ -77,6 +77,13 @@ typedef sha512_256_context_t psa_hashes_sha512_256_ctx_t;
 #include "hashes/sha256.h"
 #endif
 
+#if (IS_USED(MODULE_PSA_RIOT_HASHES_SHA_3))
+#include "hashes/sha3.h"
+
+typedef keccak_state_t psa_hashes_sha3_ctx_t;
```

Comment:
Is there a specific reason for that name?

Reply:
Do you mean `keccak_state_t`? That comes from `hashes/sha3.h`.